_docs/services/aws-elasticsearchBETA.md

```diff
@@ -89,6 +89,105 @@ For customers that would like to import or export their Elasticsearch data, this
 
  cloud.gov does offer a code sample repository on Github - [aws-elasticsearch-example](https://github.com/cloud-gov/aws-elasticsearch-example) that shows an example in Python on how to interact with the new ES service using signed headers.  Our customers are encouraged to submit PRs of other examples to share with fellow customers.
 
+## Migration from Kubernetes Elasticsearch 5.6 to AWS Elasticsearch 7.4
+Migration is not recommended and very expertimental due to the nature of Elasticsearch indexes. This is only available for customers who ask for it. 
```

Comment:
How about:

```
This content is only applicable to customers using the pre-AWS Elasticsearch service on cloud.gov during the transition period before that service is fully deprecated in April 2021. 

We recommend that you build new document indices in AWS Elasticsearch from your source documents instead of migrating the data and indices from the existing service to the new service. The migration is experimental could result in more time and effort than a rebuild would entail. 
```
If you are moving from the existing